README.md

```diff
@@ -29,6 +29,22 @@ To create the ClusterClass and it's Templates with this runtime extension enable
 kubectl apply --server-side -f examples/capi-quick-start/docker-cluster-class.yaml
 ```
 
+If creating an AWS cluster using the files in `examples/`, you will also need to create a secret with your AWS credentials:
```

Comment:
I think we should put this in the docs folder (and hence the github pages site) instead of the README. We can do that later though as we improve the docs this way.